modules/apps/headless/headless-builder/headless-builder-api/src/main/java/com/liferay/headless/builder/application/APIApplication.java

```diff
@@ -158,9 +160,9 @@ private PropertyType(String value) {
 
 		public enum Type {
 
-			AGGREGATION, ATTACHMENT, BOOLEAN, DATE, DATE_TIME, DECIMAL, INTEGER,
-			LONG_INTEGER, LONG_TEXT, MULTISELECT_PICKLIST, PICKLIST,
-			PRECISION_DECIMAL, RICH_TEXT, TEXT
+			AGGREGATION, ARRAY, ATTACHMENT, BOOLEAN, DATE, DATE_TIME, DECIMAL,
+			INTEGER, LONG_INTEGER, LONG_TEXT, MULTISELECT_PICKLIST, OBJECT,
```

Comment:
Not sure if OBJECT in this context is correct, as we are already managing object definitions, it could be confussing. What about SINGLE_CONTAINER and ARRAY_CONTAINER?